dev/package-examples/default-1.0.0/manifest.yml

```diff
@@ -0,0 +1,23 @@
+name: default
+title: Default
+description: >
+  The default package contains assets which are needed for the basic setup of the stack.
+
+  It contains the default ILM policies.
+version: 1.0.0
+categories: []
+release: ga
+# The user should not see this package and not be able to install it
+visible: false
```

Comment:
@jfsiii @skh Any thoughts on a param naming for packages which are internal and should not be shown in the UI?

Reply:
`private` and `internal` are also options

Reply:
went for `internal: true`